dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -18,6 +18,14 @@ def wait_until(timeout = DEFAULT_WAIT_TIMEOUT)
   end
 end
 
+def wait_until_interactable(timeout = DEFAULT_WAIT_TIMEOUT)
```

Comment:
This will wait until an element is 'clickable' which is necessary because the dropdown menu has an animation and for a small amount of time, the menu item is not tangible. Without this the tests are flaky.

I didn't want to add it to the general `wait_until` since I did not want to accidentally affect other tests.